spec/requests/api/v1/targets/create_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
```

Comment:
you should check if you have the `.rspec` file. If you do, then you shouldn't need this 😄